docker-for-windows/release-notes.md

```diff
@@ -20,6 +22,67 @@ for Windows](install.md#download-docker-for-windows).
 
 ## Stable Releases of 2019
 
+## Docker Desktop Community 2.1.0.0
+2019-07-31
+
+[Download](https://hub.docker.com/?overlay=onboarding)
+
+### Upgrades
+
+* [Docker 19.03.1](https://github.com/docker/docker-ce/releases/tag/v19.03.1)
+* [Docker Compose 1.24.1](https://github.com/docker/compose/releases/tag/1.24.1)
+* [Kubernetes 1.14.3](https://github.com/kubernetes/kubernetes/releases/tag/v1.14.3)
+* [Compose on Kubernetes 0.4.23](https://github.com/docker/compose-on-kubernetes/releases/tag/v0.4.23)
+* [Docker Machine 0.16.1](https://github.com/docker/machine/releases/tag/v0.16.1)
+* [linuxkit v0.7](https://github.com/linuxkit/linuxkit/releases/tag/v0.7)
+* Linux Kernel 4.9.184
+* [Kitematic 0.17.6](https://github.com/docker/kitematic/releases/tag/v0.17.6)
+* [Qemu 4.0.0](https://github.com/docker/binfmt) for cross compiling for ARM
+* [Alpine 3.10](https://alpinelinux.org/posts/Alpine-3.10.0-released.html)
+* [Docker Credential Helpers 0.6.3](https://github.com/docker/docker-credential-helpers/releases/tag/v0.6.3)
+
+### New
+
+* Selecting the ‘Experimental features’ checkbox in the Daemon, **Settings** menu turns on experimental features for Docker daemon and Docker CLI.
+* Docker Desktop now checks for stored credentials at startup before attempting to mount any shared drives. This prompts users to reenter the credentials if they are invalid.
+
+### Experimental
+
+> Experimental features provide early access to future product functionality. These features are intended for testing and feedback only as they may change between releases without warning or can be removed entirely from a future release. Experimental features must not be used in production environments. Docker does not offer support for experimental features.
+
+Docker Desktop Community 2.1.0.0 contains the following experimental features:
+
+* Docker App: Docker App is a CLI plugin that helps configure, share, and install applications. For more information, see [Working with Docker App](/app/working-with-app/).
+* Docker Buildx: Docker Buildx is a CLI plugin for extended build capabilities with BuildKit. For more information, see [Working with Docker Buildx](/buildx/working-with-buildx/).
+
+### Bug fixes and minor changes
+
+* Renamed the product from **Docker for Windows** to **Docker Desktop**.
+* Introduced a new user interface for the Docker Desktop **Settings** menu. The **Restart** and **Reset** options are now available on the **Troubleshoot** menu.
```

Comment:
Same as on mac, only the diagnose menuitem has been renamed troubleshooting

Reply:
Fixed in the latest commit.